opencti-platform/opencti-front/src/private/components/widgets/WidgetCreationParameters.tsx

```diff
@@ -88,6 +89,30 @@ const WidgetCreationParameters: FunctionComponent<WidgetCreationParametersProps>
     );
   };
 
+  const availableColumns: WidgetColumn[] = [
+    { attribute: 'entity_type' },
+    { attribute: 'relationship_type' },
+    { attribute: 'from.entity_type' },
+    { attribute: 'from.relationship_type' },
+    { attribute: 'to.entity_type' },
+    { attribute: 'to.relationship_type' },
+    { attribute: 'created_at' },
+    { attribute: 'createdBy.name' },
+    { attribute: 'objectMarking' },
+  ];
+  const columns = dataSelection[0]?.columns ?? availableColumns;
```

Comment:
components of dataSelection are handled more generally with index i 

cf code after this:
![image](https://github.com/user-attachments/assets/584ad0d4-07c3-4709-a89e-1b9db062c3e0)